techniques/systemSettings/misc/variableFromJsonFile/1.0/metadata.xml

```diff
@@ -44,7 +44,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     <SECTION name="Variable">
       <INPUT>
         <NAME>NAMESPACE</NAME>
-        <DESCRIPTION>Variable namespace, the prefix that will be used for variables definition</DESCRIPTION>
+        <DESCRIPTION>Variable namespace</DESCRIPTION>
+        <LONGDESCRIPTION>Variable namespace, the prefix that will be used for variables definition. Example: generic_variable_definition</LONGDESCRIPTION>
```

Comment:
I still think this should be a real world example, not "generic_variable_definition" which is a really terrible variable name.